.github/workflows/ci.yml

```diff
@@ -17,15 +17,19 @@ jobs:
       with:
         go-version: ${{ env.GO_VERSION }}
     - name: golangci-lint
-      uses: golangci/golangci-lint-action@v2
+      uses: golangci/golangci-lint-action@v3
       with:
         # Optional: version of golangci-lint to use in form of v1.2 or v1.2.3 or `latest` to use the latest version
-        version: v1.29
-        args: -v --concurrency 2 --timeout 10m
+        version: v1.45
+        # Optional: golangci-lint command line arguments.
+        args: |
+          --verbose
         # Optional: show only new issues if it's a pull request. The default value is `false`.
         only-new-issues: true
-        # Optional: if set to true then the action will use pre-installed Go.
-        skip-go-installation: true
```

Comment:
v3 of the GitHub Action removes the need for this config https://github.com/golangci/golangci-lint-action#compatibility